tests/distributed/test_index_query/flow.yml

```diff
@@ -1,15 +1,16 @@
 !Flow
+version: '1'
 with:
   read_only: true
   rest_api: true
   port_expose: $JINA_GATEWAY_REST_PORT
 pods:
-  encoder:
+  - name encoder:
```

Comment:
are we sure the syntax is not - name: encoder?

Reply:
@JoanFM my bad.. updating